docs/development/workflow/get_devel_version.rst

```diff
@@ -15,22 +15,22 @@ Try the development version
     `astropy forums <http://www.astropy.org/help.html>`_ to get help.
 
 
-Trying out the development version of Astropy is useful in three ways:
+Trying out the development version of `Astropy`_ is useful in three ways:
```

Comment:
I don't think we need to make all instances of 'Astropy' into links - in fact I think here we should be using the lowercase astropy (the core package) not Astropy (the project)

Reply:
Yup, agreed. Applying the distinction between Astropy the project and astropy the core package in the new commit 51d5460.

Reply:
Squashed all commits into one in c1a4ebc.